src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -1646,7 +1655,9 @@ internal static void AddSerializationMembers(CodeClass model, bool includeAdditi
                 Kind = CodeClassKind.QueryParameters,
                 Documentation = new()
                 {
-                    Description = (operation.Description ?? operation.Summary).CleanupDescription(),
+                    Description = (operation.Description is string description && !string.IsNullOrEmpty(description) ?
+                                    description :
+                                    operation.Summary).CleanupDescription(),
```

Comment:
Can safely we assume that operation.Summary is non-null and not empty here?

Reply:
same as the other instance, resolving